cli/import-jdl.mjs

```diff
@@ -16,22 +16,23 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-const chalk = require('chalk');
-const fs = require('fs');
-const _ = require('lodash');
-const path = require('path');
-const pluralize = require('pluralize');
-const { fork: forkProcess } = require('child_process');
-
-const EnvironmentBuilder = require('./environment-builder.cjs');
-const { CLI_NAME, GENERATOR_NAME, logger, toString, printSuccess, getOptionAsArgs } = require('./utils.cjs');
-
-const { packageJson: packagejs } = require('../lib/index.cjs');
-const statistics = require('../generators/statistics.cjs');
-const { JHIPSTER_CONFIG_DIR } = require('../generators/generator-constants.cjs');
-
-const jhipsterCli = require.resolve('./cli.cjs');
-const { writeConfigFile } = require('./export-utils.cjs');
+import chalk from 'chalk';
+import fs from 'fs';
+import _ from 'lodash';
+import path from 'path';
+import pluralize from 'pluralize';
+import { createRequire } from 'module';
+import { fork as forkProcess } from 'child_process';
+import EnvironmentBuilder from './environment-builder.mjs';
+import { CLI_NAME, GENERATOR_NAME, logger, toStringJoinArgs, printSuccess, getOptionAsArgs } from './utils.mjs';
+import { packageJson as packagejs } from '../lib/index.cjs';
+import statistics from '../generators/statistics.cjs';
+import { JHIPSTER_CONFIG_DIR } from '../generators/generator-constants.mjs';
+import { writeConfigFile } from './export-utils.cjs';
+
+const require = createRequire(import.meta.url);
+
+const jhipsterCli = require.resolve('./cli.mjs');
```

Comment:
No solution to use `import()` here?

Reply:
I will switch to path.join. Import.resolve is experimental.